### DISCUSSION TITLE
Zircolite in pypi

### DISCUSSION THREAD

**Discussion by kenleejl** [wagga40/Zircolite#67] [Ideas]
Can you upload zircolite to pypi? Zircolite-dev would be useful!

**Comment by wagga40** [comment]
Yes, I could, it is a great idea. 
It will require me some work to package for PyPI so don’t expect it tomorrow but I’ll work on it.

**Reply by kenleejl** [reply]
Thanks! looking forward to the PyPI package!

**Reply by kenleejl** [reply]
@wagga40 Hi any updates?

**Reply by wagga40** [reply]
Hi, 

Zircolite is available on pypi [here](https://pypi.org/project/zircolite/).

I am not happy with the package right now, I have to work on it but it works.

Since it is a Pypi release, it is missing : 
- config/mappings file, you will have to provide it (default one is here : https://github.com/wagga40/Zircolite/blob/master/config/fieldMappings.json)
- rulesets will have to be generated or taken from [here](https://github.com/wagga40/Zircolite-Rules)
- package (Gui) file so "--package" will not work as is (you have to recreate "gui/zircogui.zip" in the working directory)

Therefore, the command line is : 

```shell
zircolite -e /Users/wagga/Desktop/samples.evtx -r rules_windows_sysmon_full.json -c fieldMappings.json
```
